deployment/ansible/roles/raster-foundry.nexus/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Install PyYAML
+  pip: name=pyyaml version={{ pyyaml_version }}
+
+- name: Create directory for persistant Nexus data
+  file: name=/opt/nexus-data state=directory
+        mode=755 owner=200 group=200
```

Comment:
What is the significance of owner and group 200 here?

Reply:
See: https://hub.docker.com/r/sonatype/nexus3/

>A persistent directory, `/nexus-data`, is used for configuration,
logs, and storage. This directory needs to be writable by the Nexus
process, which runs as UID 200.